theme/static/js/hs-vue/profile-link-cmp.js

```diff
@@ -4,21 +4,21 @@ Vue.component('profile-link', {
     data: function () {
         return {
             user: {
-                user_type: "initial",
-                access: "initial",
-                id: "initial",
-                pictureUrl: "initial",
-                best_name: "initial",
-                user_name: "initial",
-                can_undo: "initial",
-                email: "initial",
-                organization: "initial",
-                title: "initial",
-                contributions: "initial",
+                user_type: "initial_link",
```

Comment:
Why does vue need an initial data object?  Why isn't this built as the data comes in?

Reply:
Vue needs a skeleton in order to build the instance and set its "virtual DOM". The way this component works, data for this object is updated when a user clicks on a link. So, a placeholder object needs to be passed to Vue so it can load.

I will swap these initial values to `null` to make it cleaner.

Reply:
This comment also seems to be referencing an outdated version of the files.